src/common/tiles/BigTile/index.js

```diff
@@ -18,45 +19,57 @@ import {
   VoteAmount,
   Article,
 } from "./styled";
-import poster from "./images/poster.png";
 
-export const BigTile = () => (
-  <Container>
-    <Image src={poster} />
-    <TileContainer>
-      <Title>Mulan</Title>
-      <Year>2020</Year>
-      <SubHeaderContainer>
-        <SubHeader>
-          <SubHeaderTitle>Production:</SubHeaderTitle>
-          <SubHeaderInformation>
-            China, United States of America
-          </SubHeaderInformation>
-        </SubHeader>
-        <SubHeader>
-          <SubHeaderTitle>Release date:</SubHeaderTitle>
-          <SubHeaderInformation>24.10.2020</SubHeaderInformation>
-        </SubHeader>
-      </SubHeaderContainer>
-      <GenreContainer>
-        <Genre>Action</Genre>
-        <Genre>Drama</Genre>
-        <Genre>Adventure</Genre>
-      </GenreContainer>
-      <VotesContainer>
-        <RatesContainer>
-          <StarIcon />
-          <ActualRating>7,8</ActualRating>
-          <MaxRating>/ 10</MaxRating>
-        </RatesContainer>
-        <VoteAmount>335 votes</VoteAmount>
-      </VotesContainer>
-    </TileContainer>
-    <Article>
-      A young Chinese maiden disguises herself as a male warrior in order to
-      save her father. Disguises herself as a male warrior in order to save her
-      father. A young Chinese maiden disguises herself as a male warrior in
-      order to save her father.
-    </Article>
-  </Container>
-);
+export const BigTile = ({
+  id,
+  poster_path,
+  title,
+  release_date,
+  rating,
+  votes,
+  countries,
+  article,
+}) => {
+  return (
+    <MainContainer key={id} id={id}>
+      <Image src={`${imageBaseUrl}/w342${poster_path}`} />
+      <TileContainer>
+        <BigTileTitle>{title}</BigTileTitle>
+        <Year>{release_date}</Year>
+        <SubHeaderContainer>
+          <SubHeader>
+            {countries.length > 0 ? (
+              <SubHeaderTitle>Production:</SubHeaderTitle>
+            ) : null}
+            {countries.map(({ name }) => (
+              <>
+                <SubHeaderInformation>
+                  {name}
+                  {","}
+                </SubHeaderInformation>
+              </>
+            ))}
+          </SubHeader>
+          <SubHeader>
+            <SubHeaderTitle>Release date:</SubHeaderTitle>
+            <SubHeaderInformation>{release_date}</SubHeaderInformation>
```

Comment:
Tu tak samo, jest zła data. Tylko nie wiem czy uda nam się to zmienić tak łatwo :(

Reply:
Dat nie ruszalem, bo chyba sie umawialismy na 1 callu, ze narazie daty zostawimy pelne, albo cos zle zrozumialem :)

Reply:
Bardzo dobrze zrozumiales :) na daty przyjdzie czas :) chyba pomysl z new Date byl w planach